dali/test/python/operator/test_random_resized_crop.py

```diff
@@ -20,7 +20,9 @@
 
 
 def close(a, b):
-    return abs(a - b) < 1e-5 or abs(a - b) < abs(a) + abs(b) * 1e-6
```

Comment:
I can't wrap my head around the `or` part. It is always true if a >= b >= 0.

Reply:
actually I don't really understand the need for the second expression :/

Reply:
a=10, b=11
abs(a-b) -> 1
abs(a) + abs(b) * 1e-6 -> 10 + 11*1e-6 -> 10.000011
1 < 1e-5 -> false
10 < 10.000011 -> true

Reply:
a=10, b=30
abs(a-b) = 20
abs(a) + abs(b) * 1e-6 -> 10.00003
10 < 1e-5 -> false
20 < 10.00003 -> false

Reply:
Yes, it should have been `(abs(a) + abs(b)) * 1e-6`